src/main/java/core/basesyntax/exeptions/InvalidDataException.java

```
@@ -0,0 +1,6 @@
+package core.basesyntax.exeptions;
+public class InvalidDataException extends RuntimeException {
+    public InvalidDataException(String message) {
+        super(message);
```

Add cause-preserving constructors. The project checklist requires rethrowing exceptions with the original cause: "Don't ignore exceptions in try-catch blocks ... Let's rethrow a RuntimeException with an informative message and exception object." Add constructors `InvalidDataException(String message, Throwable cause)` and `InvalidDataException(Throwable cause)` so callers can wrap and preserve underlying exceptions (e.g., NumberFormatException or IOException). See current class (only single-message constructor exists)  .